packages/stable/src/api/functions/functionCallsApi.ts

```diff
@@ -0,0 +1,69 @@
+import { BaseResourceAPI, type HttpResponse } from '@cognite/sdk-core';
+import type {
+  FunctionCall,
+  FunctionCallListScope,
+  FunctionCallListWithCursorResponse,
+  FunctionCallLogResponse,
+  FunctionCallRequest,
+  FunctionCallResponse,
+  FunctionCalledResponse,
+} from './types.gen';
+
+export class FunctionCallsApi extends BaseResourceAPI<FunctionCall> {
+  /**
+   * @hidden
+   */
```

Comment:
Why hidden?

Reply:
Snippets or samples, one of those tools was failing

Reply:
Ok, we should look into that. `@hidden` will hide this endpoint from the documentation.
I can look into it.

Reply:
I just tested and this works:
```ts
  /**
   * [List function calls](https://api-docs.cognite.com/20230101/tag/Function-calls/operation/postFunctionsCall)
   *
   * ```js
   * client.functions.call(123, { nonce: 'abc' } )
   * ```
   */
```
🤷